core/docker/docker-compose.yaml

```diff
@@ -59,12 +61,12 @@ services:
       - broker
       - schema-registry
     ports:
-      - 18083:18083
```

Comment:
why did you change the ports? I honestly don't remember why they were not standard... but just be sure it doesn't break smg else :) fine for me  @moliele

Reply:
Once I updated the images versions I faced with this error:
curl: (56) Recv failure: Connection reset by peer.

It was related to not using the standard ports. 

That's the reason why they have been changed, I changed them everywhere, but I will check again in order to avoid breaking something.